packages/app-desktop/gui/NoteEditor/utils/useFormNote.ts

```diff
@@ -58,12 +59,15 @@ function resourceInfosChanged(a: ResourceInfos, b: ResourceInfos): boolean {
 	return false;
 }
 
+const getCompareKey = (a: FormNote) => `${a.body}|${a.id}|${a.title}|${a.parent_id}|${a.is_todo}`;
+const isNoteEqual = (a: FormNote, b: FormNote) => getCompareKey(a) === getCompareKey(b);
```

Comment:
I think you should only compare updated_time. In fact maybe that's all you need to pass to the component, instead of all the notes.

Reply:
Unfortunately, FormNote does not have the `updated_time` property. So it can not be used for comparison.

However, note that this is just my humble try to improve performance and check if there is any change in content to not always load the note. 

I can also skip the check if you want, it's not super important.

What do you think?